riot/lib/gen_server.ml

```diff
@@ -31,6 +33,11 @@ let call : type res. Pid.t -> res req -> res =
       | None -> failwith "bad message")
   | _ -> failwith "unexpected message"
 
+let cast : type res. Pid.t -> res req -> unit =
```

Comment:
**suggestion**. I think we can write this without a locally abstract type since the `Cast` constructor would already take any type as input.